indico/modules/events/registration/forms.py

```diff
@@ -423,3 +423,9 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    name = StringField(_('Name'), [DataRequired()])
+    color = StringField(_('Color'), [DataRequired()], widget=ColorInput())
+    fields = ['name', 'color']
```

Comment:
not needed; `form_fields` works fine without specifying a list of fields and in almost all cases that's enough (no custom order, omitting some fields, etc)